### PR TITLE
Clean up replayer

### PR DESCRIPTION
**Pull request by jakeswenson**

### CODE REVIEW

File: src/main/java/com/godaddy/icfp2017/Application.java [hunk 1]
```diff
@@ -45,32 +45,48 @@ public static void main(String[] args) throws Exception {
...
+      gameDriver.run();
+
+      if(shouldCapture) {
+        gameDriver.dumpCapture(System.err);
+      }
+
+    }
+    finally {
+      if (gameDriver != null) {
+        gameDriver.close();
```
Comment (jakeswenson) [inline src/main/java/com/godaddy/icfp2017/Application.java:84]: the game now shutdown correctly

File: src/main/java/com/godaddy/icfp2017/services/GameLogic.java [hunk 2]
```diff
@@ -205,7 +210,7 @@ public GameplayP2S move(final GameplayS2P move) {
 
     // initialize the response
     final GameplayP2S response = new GameplayP2S();
-    response.setPass(pass); // todo change this to setClaim
+//    response.setPass(pass); // todo change this to setClaim
```
Comment (jakeswenson) [inline src/main/java/com/godaddy/icfp2017/services/GameLogic.java:215]: not sure why we were always setting pass...

Comment (kercheval) [reply src/main/java/com/godaddy/icfp2017/services/GameLogic.java:215]: Just remove the line completely

**Comment by kercheval** [comment]
👍

**Pull request merged**